model/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -110,7 +111,8 @@ case class ExecutionServiceWorkflowOptions(
   monitoring_image_script: Option[String] = None
 )
 
-// current possible backends are "JES" and "PAPIv2" but this is subject to change in the future
+// current possible backends are "PAPIv2-beta", "PAPIv2-CloudNAT", and "GCPBatch",
+// but this is subject to change in the future
```

Comment:
I don't think "subject to change" adds any new info
```suggestion
// current possible backends are "PAPIv2-beta", "PAPIv2-CloudNAT", and "GCPBatch"
```

Reply:
Agree with Adam's suggestion and also think it would be good to note that PAPIv2-beta is not actually in use right now.